api/v1beta1/nutanixmachine_types.go

```diff
@@ -65,8 +65,8 @@ type NutanixImageLookup struct {
 	Format *string `json:"format,omitempty"`
 	// ImageLookupBaseOS is the name of the base operating system to use for
 	// image lookup.
-	// +kubebuilder:validation:Required
-	BaseOS *string `json:"baseOS"`
+	// +kubebuilder:validation:MinLength:=1
+	BaseOS string `json:"baseOS"`
```

Comment:
In context of discussion yesterday, if we extend lookup with categories, this could field be optional too. so, let's not treat it like a required field

Reply:
This becomes tricky then, i think we should leave this as required for now and add it to be optional when we support category look up